logging/bin/secrets-include.sh

```diff
@@ -8,101 +8,105 @@
 
 function create_secret_from_file {
 
-  file=$1
-  secret_name=$2
-  label=$3
-
-  if [ -z "$(kubectl -n $LOG_NS get secret $secret_name -o name 2>/dev/null)" ]; then
-
-    log_debug "Creating secret [$secret_name]"
-
-    if [ -f "$USER_DIR/logging/$file" ]; then filepath=$USER_DIR/logging
-    elif [ -f "logging/opensearch/$file" ]; then filepath=logging/opensearch
-    else
-      log_error "Could not create secret [$secret_name] because file [$file] could not be found"
-      return 9
-    fi
-
-    if [ "$(kubectl -n $LOG_NS create secret generic $secret_name --from-file=$filepath/$file)" == "secret/$secret_name created" ]; then
-      log_verbose "Created secret for OpenSearch config file [$file]"
-
-      if [ "$label" != "" ]; then
-         log_debug "Applying label [$label] to newly created secret [$secret_name]"
-         kubectl -n $LOG_NS label secret  $secret_name $label
-      fi
-
-      return 0
+    file=$1
+    secret_name=$2
+    label=$3
+
+    if [ -z "$(kubectl -n "$LOG_NS" get secret "$secret_name" -o name 2> /dev/null)" ]; then
+
+        log_debug "Creating secret [$secret_name]"
+
+        if [ -f "$USER_DIR/logging/$file" ]; then
+            filepath=$USER_DIR/logging
+        elif [ -f "logging/opensearch/$file" ]; then
+            filepath=logging/opensearch
+        else
+            log_error "Could not create secret [$secret_name] because file [$file] could not be found"
+            return 9
+        fi
+
+        if [ "$(kubectl -n "$LOG_NS" create secret generic "$secret_name" --from-file="$filepath"/"$file")" == "secret/$secret_name created" ]; then
+            log_verbose "Created secret for OpenSearch config file [$file]"
+
+            if [ -n "$label" ]; then
+                log_debug "Applying label [$label] to newly created secret [$secret_name]"
+                # shellcheck disable=SC2086
+                # quoting $label results in invalid kubectl syntax
```

Comment:
I wonder if we should have _require_ an explanation when disabling any shellcheck directive in the code?  Adding an explanation after the disable directive (e.g. `#shellcheck disable=SC2086 quoting $label results in invalid kubectl syntax`) is NOT valid, so I had to add a 2nd comment.   The Shellcheck doc shows including both comments on the same line as acceptable alternative, i.e. `#shellcheck disable=SC2086    #quoting $label results in invalid kubectl syntax`.  That might be cleaner/more readable.  Thoughts?